README.md

```diff
@@ -34,7 +34,7 @@ memory-serve is designed to work with [axum](https://github.com/tokio-rs/axum)
 ## Usage
 
 Provide a relative path to the directory containing your static assets
-to the [`load_assets!`] macro. This macro creates a data structure intended to
+usinf the `ASSET_PATH` environment variable. This macro creates a data structure intended to
```

Comment:
```suggestion
using the `ASSET_PATH` environment variable. This macro creates a data structure intended to
```

Reply:
Fixed in [638807b](https://github.com/tweedegolf/memory-serve/pull/15/commits/638807b881dbd0ae70e1a6ef47198ec7ccffe17c)